commit b3fb95d1fd00490a78b723a825e76ecafbf3b27a
Author: Noah Watkins <noahwatkins@gmail.com>
Date:   Sun Oct 13 11:20:16 2024 -0700

    Add fusion dependency to iterator
    
    external/rules_boost~~non_module_dependencies~boost/libs/iterator/include/boost/iterator/zip_iterator.hpp:20:10: error: 'boost/fusion/adapted/boost_tuple.hpp' file not found [clang-diagnostic-error]
       20 | #include <boost/fusion/adapted/boost_tuple.hpp> // for backward compatibility

diff --git a/boost.BUILD b/boost.BUILD
index 588030c..dbaf6ff 100644
--- a/boost.BUILD
+++ b/boost.BUILD
@@ -1096,6 +1096,7 @@ boost_library(
     name = "iterator",
     deps = [
         ":detail",
+        ":fusion",
         ":static_assert",
         ":type_traits",
         ":utility",
